modules/vf-graphql-holochain/connection.ts

```diff
@@ -139,7 +139,11 @@ const HOLOHASH_PREFIX_AGENT = [0x84, 0x20, 0x24]   // uhCAk
 
 const serializedHashMatchRegex = /^[A-Za-z0-9_+\-/]{53}={0,2}$/
 const idMatchRegex = /^[A-Za-z0-9_+\-/]{53}={0,2}:[A-Za-z0-9_+\-/]{53}={0,2}$/
-const stringIdRegex = /^\w+?:[A-Za-z0-9_+\-/]{53}={0,2}$/
+// something like
+// $:uhC0k1mcUqQIbtT0mkdTldhBaAvR6KlKxIV2IYwJemHt-NO92uXG5
+// or kg:uhC0k1mcUqQIbtT0mkdTldhBaAvR6KlKxIV2IYwJemHt-NO92uXG5
+// but not 9:uhC0k1mcUqQIbtT0mkdTldhBaAvR6KlKxIV2IYwJemHt-NO92uXG5 (i.e. no digits in the id)
+const stringIdRegex = /^\D+?:[A-Za-z0-9_+\-/]{53}={0,2}$/
```

Comment:
I'm unsure what the `_+\-/` section is meant to capture, but since that hasn't changed in this commit I'm assuming its okay.

Reply:
it's just the base64 encoding that gets done, I believe those are valid characters.